google/cloud/storage/examples/storage_bucket_samples.cc

```diff
@@ -325,6 +326,33 @@ void PatchBucketStorageClassWithBuilder(google::cloud::storage::Client client,
   (std::move(client), bucket_name, storage_class);
 }
 
+void GetBucketClassAndLocation(google::cloud::storage::Client client, int& argc,
+                               char* argv[]) {
+  if (argc != 2) {
+    throw Usage{"get-bucket-class-and-location <bucket-name>"};
+  }
+  auto bucket_name = ConsumeArg(argc, argv);
+  // [START storage_get_bucket_class_and_location]
+  namespace gcs = google::cloud::storage;
+  using ::google::cloud::StatusOr;
+  [](gcs::Client client, std::string bucket_name) {
+    StatusOr<gcs::BucketMetadata> bucket_metadata = client.GetBucketMetadata(
+        bucket_name, gcs::Fields("name,location,storageClass"));
+
+    if (!bucket_metadata) {
+      throw std::runtime_error(bucket_metadata.status().message());
+    }
+
+    std::cout << "Bucket " << bucket_metadata->name()
+              << " default storage class is "
+              << bucket_metadata->storage_class()
+              << ", and the default location is " << bucket_metadata->location()
```

Comment:
"bucket location" instead of "default location"

Reply:
Fixed.